static/js/components/tiles/map_tile.tsx

```diff
@@ -196,12 +204,21 @@ export function MapTile(props: MapTilePropType): JSX.Element {
     !!zoomParams && !!mapChartData && _.isEqual(mapChartData.props, props);
 
   useEffect(() => {
-    if (_.isEmpty(mapChartData) || !_.isEqual(mapChartData.props, props)) {
+    if (
+      _.isEmpty(mapChartData) ||
+      !_.isEqual(mapChartData.props, props) ||
+      !_.isEqual(mapChartData.dateOverride, dateOverride)
+    ) {
       loadSpinner(props.id);
       (async () => {
-        const data = await fetchData(props);
-        if (data && props && _.isEqual(data.props, props)) {
-          setMapChartData(data);
+        setIsLoading(true);
```

Comment:
Good call.
Explore page: https://screenshot.googleplex.com/8EVJQ8CpDbTnerz
Map tool: https://screenshot.googleplex.com/3Yc8TW5UUQ6egLp

the map tool is showing two spinners, but looks decent. what do you think?

Reply:
I think the double spinner on Map tool is ok if they also finish loading at the same time. Would be frustrating to have one finish spinning but still need to wait on the other.

Reply:
hm, I feel like the double spinner makes it look a little unpolished like we forgot to remove one, but maybe that's ok.. @miss-o-soup what do you think?

Aside from the way the double spinner looks, thinking more about the code, I think it feels a little confusing to:
- always show the loading message when fetching data because there is a prop that's supposed to control whether or not to show loading: https://github.com/datacommonsorg/website/blob/master/static/js/components/tiles/map_tile.tsx#L115-L116
- have two separate loading treatments for one tile

Some ideas:
- Have map tool use the same new loading treatment & can remove the old loading code including the prop that controls whether or not to show loading. One concern is that this won't be consistent with the loading treatment on the other vis tools (maybe update the loading treatment for those tools as well?) 
- update the original `showLoadingSpinner` prop to something like `onLoading` or something to allow the caller to customize how to handle loading. Then default loading behavior is the new `Loading..` you're adding, but if this prop is set, don't do the default loading behavior and do the customized behavior

Reply:
Removed the duplicate loading indicator from the map viz tool (Removed `showLoadingSpinner` prop entirely from the map component). 

Now looks like: 

![6RxyRnxJuhHWnzt](https://github.com/datacommonsorg/website/assets/13766/d8dc88f0-c5b5-48b3-836e-2c68b6b6adb7)

Ill make the loading treatment consistent across the rest of the visualization tools in a follow up PR.